framework/src/main/java/org/checkerframework/framework/util/dependenttypes/DependentTypesHelper.java

```diff
@@ -440,7 +440,8 @@ public void standardizeForMethodSignature(
         JavaExpressionContext context =
                 JavaExpressionContext.buildContextForMethodDeclaration(
                         methodDeclTree, enclosingType, factory.getChecker());
-        standardizeDoNotUseLocalScope(context, pathToMethodDecl, atm, removeErroneousExpressions);
+        standardizeAtm(
+                context, pathToMethodDecl.getParentPath(), atm, false, removeErroneousExpressions);
```

Comment:
I think you want to pass `true` here?  I think you are trying change `localScope` to `true` in all places so that it can be removed right?  (Or I'm I confused?)

Reply:
There is a branch `remove-useLocalScope-2` in which all uses of `useLocalScope` have been eliminated.  I was trying to minimize this pull request to make it easier to review, but could make the full pull request if you want to see all the changes together.